soak_bench_scripts/soak.sh

```diff
@@ -38,4 +39,10 @@ rm target/release/integration_tests-*.d
 printf "Starting soak tests...\n"
 
 # Loop the test forever until it hits an error
-loop -q target/release/integration_tests-* |& tee -a -i -p --output-error=warn $LOGFILE
+os=$(uname)
```

Comment:
Better to be consistent when naming variables.  Others are all `SCREAMING_SNAKE_CASE`, so this should be `OS`

Reply:
Good point, will update.